src/main/java/org/diego/tutorial/car/databases/jpa/JPAImplCar.java

```diff
@@ -20,9 +20,11 @@
 	 * @return List of cars from the country searched
 	 */
 	public List<Car> getAllCarsFromCountry(String country){
-		String countryLowerCase = country.toLowerCase();
-		
-		String query = "SELECT car FROM Car car WHERE car.country='" + countryLowerCase + "'";
+		String query = "SELECT car "
+					+ "FROM Car car "
+					+ "WHERE car.country = (SELECT country "
+											+ "FROM Country country "
+											+ "WHERE country.countryName = '" + country  + "')";

```

Comment:
I would do this better using criteriaQuery

Reply:
done